src/Symfony/Bridge/Doctrine/DataCollector/DoctrineDataCollector.php

```diff
@@ -77,7 +77,7 @@ public function getConnections(): array
 
     public function getQueryCount(): int
     {
-        return array_sum(array_map('count', $this->data['queries']));
+        return array_sum(array_map(count(...), $this->data['queries']));
```

Comment:
The first thing that comes to mind is that this could be much better for static analysis tools.
But beyond that, by generalizing this syntax, one of the big plus points is that we can make sure there are no errors or typo when we use a function, where strings might let innatention errors slip through.